python/scripts/archive_repositories.py

```diff
@@ -77,19 +77,17 @@ def get_config_for_organization(github_organization_name: str) -> tuple[str, lis
         f"Unsupported Github Organization Name [{github_organization_name}]")
 
 
-def archive_inactive_repositories_by_date_and_type(github_token: str, organization_name: str, archive_date: datetime,
+def archive_inactive_repositories_by_date_and_type(github_service: GithubService, archive_date: datetime,
                                                    allow_list: list[str],
-                                                   repo_type_to_archive: str):
-    moj_gh = MojGithub(org=organization_name, org_token=github_token)
-
+                                                   repository_type: str):
     repos = [
         repo
-        for repo in moj_gh.get_unarchived_repos(repo_type_to_archive)
+        for repo in github_service.get_repositories_to_consider_for_archiving(repository_type)
```

Comment:
This seems absurdly long. How about `get_repositories_for_archiving` and leaving it up to the process for consideration?

Reply:
Changed up a little bit since this comment, so I'll mark as resolved 👀